.rhdh/scripts/install-registry-proxy-images.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+#  Copyright (c) 2024 Red Hat, Inc.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# This script automates the installation of registry-proxy images onto an airgapped Kubernetes cluster. It pulls images locally, tags them for the local registry-proxy, pushes them into the cluster, and performs the catalog source installation.
+
+set -e
+
+# Define your registry-proxy URL (replace with your actual registry-proxy URL)
+REGISTRY_PROXY_URL="registry-proxy.engineering.redhat.com"
+
+# Define the target namespace in your cluster
+NAMESPACE="my-namespace"
+
+# List of image references (replace with your actual image references)
+IMAGE_REFERENCES=(
+  "quay.io/rhdh/iib:1.2-v4.14-x86_64"
```

Comment:
why hardcoded at version 1.2?

Reply:
You are right we don't, we only need to pull those containers locally and push them into the cluster.